package.json

```diff
@@ -51,8 +51,8 @@
   "scripts": {
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
-    "test": "react-app-rewired test --env=jsdom",
-    "eject": "react-app-rewired eject",
+    "test": "jest --silent --env=jsdom",
```

Comment:
Nope, somehow, it only silences that one polyfill error when I switched to raw jest testing. This will not affect build and development because those still use the polyfill. I think this change is necessary because now our tests take only 25 seconds average instead of 90.